Storage/Logins.swift

```diff
@@ -139,7 +139,7 @@ public class Login: Printable, LoginData, LoginUsageData, Equatable {
             let url1 = NSURL(string: value!)
 
             if url1?.host != url2?.host {
-                assertionFailure("Form submit URL domain doesn't match login's domain.")
+                // assertionFailure("Form submit URL domain doesn't match login's domain.")
```

Comment:
Heh. No. I have some data in my syncing profile that tickles this and causes crashes. I'll remove it.